test-dataset-plan.md

```diff
@@ -1,4 +1,4 @@
-# DadsBench: DAnish Domain-Specific language modelling Benchmark (DadsBench) v.1
+# DadsBench: *Da*nish *d*omain-*s*pecific language modelling *Bench*mark v.1
```

Comment:
My bad, single asterisks on Github markdown is italics, apparently, with double being bold:

```suggestion
# DadsBench: **Da**nish **d**omain-**s**pecific language modelling **Bench**mark v.1
```